support-frontend/assets/pages/[countryGroupId]/components/thankyou.tsx

```diff
@@ -196,7 +199,15 @@ export function ThankYouComponent({
 		];
 	}
 
-	const showNewspaperArchiveBenefit = abTests.newspaperArchiveBenefit.isActive;
+	/* display if either:-
```

Comment:
The logic is neat, but maybe having another way to check AB test participation muddies things somewhat. Might be simpler to just do the standard participation check here instead of `isActive`?

Reply:
I think I need to is to send links so the new Checkout Benefit and Thankyou card can be checked directly on production.  Without the need to either use Code or know how to get into the test via `isActive` on production. Thought this would be a useful feature going forwards (its easy to leave an abtest on accidently too). Maybe I missed a simpler way of doing this?

Reply:
Yep, agree have simplified to standard participation check now, thanks